### ISSUE TITLE
Jupyter notebook stops working

### ISSUE THREAD

**Issue by shug1565** [microsoft/vscode#123611]
We have written 
Issue Type: <b>Bug</b>

Jupyter notebook stops executing after the upgrade of the vscode insider.

VS Code version: Code - Insiders 1.57.0-insider (988902e0bc37c7acba82e2ddde49592d8cb11666, 2021-05-11T13:54:54.372Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Xeon(R) CPU E5-2620 0 @ 2.00GHz (24 x 2007)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: unavailable_off<br>opengl: enabled_on<br>rasterization: unavailable_off<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.95GB (22.12GB free)|
|Process Argv|--crash-reporter-id 717eb2b4-1f36-481a-a450-8f77d9e8959c|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (15)</summary>

Extension|Author (truncated)|Version
---|---|---
TabOut|alb|0.2.0
vscode-eslint|dba|2.1.20
gitlens|eam|11.4.1
prettier-vscode|esb|6.4.0
csharp|ms-|1.23.11
python|ms-|2021.6.832859966-dev
jupyter|ms-|2021.6.832913620
cpptools|ms-|1.4.0-insiders
powershell-preview|ms-|2021.4.2
mql-extension-pack|nic|0.0.4
mql-over-cpp|nic|0.0.3
mql-snippets|nic|0.1.2
autodocstring|njp|0.5.4
vscodeintellicode|Vis|1.2.13
vscode-icons|vsc|11.4.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
pythonvspyt602:30291494
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes516:30244333
pythonvspyt639:30291487
pythontb:30258533
pythonvspyt551:30291412
vspre833cf:30267465
pythonptprofiler:30281269
vshan820:30294714
pythondataviewer:30285072
vscus158:30286553
pythonvsuse255cf:30296410
vscgsv2ct:30286873
vscorehov:30295781

```

</details>

the needed data into your clipboard because it was too large to send. Please paste.

**Comment by joyceerhl** [comment]
@shug1565 👋 I'm a dev from the Jupyter extension. This sounds like a duplicate of https://github.com/microsoft/vscode-jupyter/issues/5838, which is resolved by v2021.6.832913620 of the Jupyter extension. Please ensure that you have installed the latest version of the Jupyter extension and reload VS Code when prompted. If the problem persists after reloading VS Code, please file an issue with us at https://github.com/microsoft/vscode-jupyter/issues/ Thank you and sorry for the inconvenience!

**Comment by shug1565** [comment]
Thanks for the quick response. Yes, it works now.

Best regards,
Jian
